crates/hir-def/src/path.rs

```diff
@@ -274,6 +278,12 @@ impl<'a> PathSegments<'a> {
             generic_args: self.generic_args.map(|it| it.get(..len).unwrap_or(it)),
         }
     }
+    pub fn strip_last(&self) -> PathSegments<'a> {
```

Comment:
Was wondering what this kind of function is usually called, given everything but the `head` is called usually `tail`. Haskell calls everything but the `last` element `init`, not sure I like that though